entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export GOOGLE_APPLICATION_CREDENTIALS="/usr/src/app/gcloud_key.json"
+gcloud config set project k8s-jkns-gke-soak
+gcloud config set account shanefu@k8s-jkns-gke-soak.iam.gserviceaccount.com
+gcloud config set compute/zone us-central1-f
```

Comment:
Same parameterizing on zone here (note: I would put the env var declarations at the top of the file
```suggestion
GCP_ZONE="${GCP_ZONE:-us-central1-f}"
gcloud config set compute/zone ${GCP_ZONE}
```

Reply:
done